docs/topics/downloader-middleware.rst

```diff
@@ -737,6 +737,7 @@ HttpCompressionMiddleware Settings
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 .. setting:: COMPRESSION_ENABLED
+.. setting:: RETRY_HTTP_COMPRESSION_ERRORS
```

Comment:
This line should be right above the `RETRY_HTTP_COMPRESSION_ERRORS` section instead :slightly_smiling_face:

Reply:
My bad 🤦‍♂